packages/tools/client-debugger/client-debugger-view/src/RootView.tsx

```diff
@@ -26,8 +29,10 @@ export interface RootViewProps {
  */
 export function RootView(props: RootViewProps): React.ReactElement {
 	return (
-		<MessageRelayContext.Provider value={props.messageRelay}>
-			<FluidClientDebuggers />
-		</MessageRelayContext.Provider>
+		<FluentProvider theme={flueUIThemeToUse()}>
```

Comment:
Wrapping with the theme here is unnecessary, I think, since `FluidClientDebuggers` already does it.

Reply:
you are right; I must have stumbled on some other issue; the styling is applied.